packages/docs/docs/auth/smart-scopes.md

```diff
@@ -0,0 +1,31 @@
+# SMART Scopes for Access Control
+
+SMART on FHIR’s authorization scheme uses OAuth scopes to communicate (and negotiate) access requirements. There is relevant [background information on HL7.org](http://hl7.org/fhir/smart-app-launch/STU2/scopes-and-launch-context.html).
+
+Medplum supports the `SMART App Launch 2.0.0` [standard](https://hl7.org/fhir/smart-app-launch/).
+
+Historically, apps developed with SMART Scopes were designed to be links, embedded in a traditional EHR. Once those links were clicked on an app would launch and authenticate the user, and request a certain FHIR resources, or grant rights like `offline_access`. Commonly, EHRs support "launching" from either a patient history screen or an encounter documentation screen. So when you see "launch" language in the documentation, hopefully that gives some context on what that means.
```

Comment:
I think this paragraph might be too confusing. 
What I found confusing was that this paragraph starts off saying "Historically, SMART scopes were designed for X", which implies that this is _not_ what SMART scopes are used for nowadays. But it doesn't seem to clearly address what use cases SMART scopes _are_ used for nowadays

After addressing my comment above, this probably becomes unnecessary.